utils/models/result.py

```diff
@@ -0,0 +1,287 @@
+import typing as tp
+from typing import List, Union
+
+from pydantic import BaseModel, Field, model_validator
+
+from utils.models.model_types import (
+    BalanceString,
+    EstimateGasPriceString,
+    FalseOnly,
+    GasPriceString,
+    HexString,
+    IdField,
+    JsonRPCString,
+    NeonVersionString,
+    NetVersionString,
+    NonZeroBytesString,
+    StorageString,
+    ZeroBytesString,
+)
+
+NEON_FIELDS = [
+    "neonAccountSeedVersion",
+    "neonMaxEvmStepsInLastIteration",
+    "neonMinEvmStepsInIteration",
+    "neonGasLimitMultiplierWithoutChainId",
+    "neonHolderMessageSize",
+    "neonPaymentToTreasury",
+    "neonStorageEntriesInContractAccount",
+    "neonTreasuryPoolCount",
+    "neonTreasuryPoolSeed",
+    "neonEvmProgramId",
+]
```

Comment:
was this intended to be used somewhere?

Reply:
fixed ✅  ( moved to the NeonGetEvmParamsDetails model and removed this list).